.github/workflows/release.yaml

```diff
@@ -150,7 +154,13 @@ jobs:
     - name: Install Poetry
       uses: Gr1N/setup-poetry@v9
 
+    - name: Print environment variables
+      run: env
```

Comment:
Any security concerns?

Reply:
I will remove this, added for debugging.

Reply:
Removed.